packages/eslint-plugin/docs/rules/no-empty-interface.md

```diff
@@ -38,6 +38,12 @@ interface Bar {
 interface Baz extends Foo, Bar {}
 ```
 
+### Options
+
```

Comment:
can you add json example of default config for rule:

```json
{
  "rule-name": ["error", {
    "allowSingleExtend": false
  }]
}

Reply:
Should be done